src/types/decimal.rs

```diff
@@ -62,6 +63,38 @@ impl Decimal {
         }
     }
 
+    /// Returns true if this Decimal's coefficient has a negative sign AND a magnitude greater than
+    /// zero. Otherwise, returns false. (Negative zero returns false.)
+    pub fn is_less_than_zero(&self) -> bool {
```

Comment:
🗺️ These two helper methods are used by the `Timestamp` type to see if a given `Decimal` representing a fractional number of seconds is in the range `[0, 1)`. `is_less_than_zero` is actually more broadly useful because applications will often want to know if a value is negative but _don't_ care if it's `-0` (which is negative, but not less than zero).